### PR TITLE
Don't reset played items on going live, fixes #180

### LINKED ISSUE #180: Item played state resets when the news happens
I presume it's because it treats "going live after the news" the same way as "going live after connecting."

### PR DESCRIPTION
**Pull request by mstratford**
It's a bad idea and it's hard to implement correctly.

### CODE REVIEW

File: src/mixer/the_news.ts [hunk 1]
```diff
@@ -52,8 +52,7 @@ const considerDoingTheNews = (getState: () => RootState) => async () => {
     await actuallyDoTheNews();
   } else if (state.settings.doTheNews === "while_live") {
     if (
-      state.broadcast.connectionState === "CONNECTED" ||
-      state.broadcast.connectionState === "LIVE"
+      state.broadcast.connectionState in ["CONNECTED", "GOING_LIVE", "LIVE"]
```
Comment (markspolakovs) [inline src/mixer/the_news.ts:55]:
```
> "a" in ["a","b","c"]
false
```

**Review by markspolakovs — CHANGES_REQUESTED** [review]

File: src/mixer/the_news.ts (hunk 1 above)
Comment (mstratford) [reply src/mixer/the_news.ts]: Woops

File: src/broadcast/rtc_streamer.ts [hunk 2]
```diff
@@ -120,6 +120,9 @@ export class WebRTCStreamer extends Streamer {
         this.state = "ANSWER";
         break;
       case "ACTIVATED":
+        if (!this.isActive) {
+          this.onStateChange("GOING_LIVE"); // Starting show.
+        }
```
Comment (markspolakovs) [inline src/broadcast/rtc_streamer.ts:125]: You sure about this? `isActive` gets reset on L131, and I'm fairly sure shittyserver fires a DEACTIVATED when the news happens

File: src/mixer/the_news.ts [hunk 3]
```diff
@@ -51,18 +51,19 @@ const considerDoingTheNews = (getState: () => RootState) => async () => {
...
-        selSource: number;
-        switchAudioAtMin: number;
-      }>("/nextTransition", "GET", {});
-      if (transition.autoNews) {
-        await actuallyDoTheNews();
-      }
+    switch (state.broadcast.connectionState) {
+      case "CONNECTED":
+      case "GOING_LIVE":
+      case "LIVE":
```
Comment (markspolakovs) [inline src/mixer/the_news.ts:57]: Do we need this? LIVE should happen in the same tick as GOING_LIVE, so would this cause an unnecessary AJAX request?

File: src/broadcast/rtc_streamer.ts (hunk 2 above)
Comment (mstratford) [reply src/broadcast/rtc_streamer.ts:125]: Gah, in which case, maybe I'd rather just make this manually reset from the nav menu

**Pull request merged**